prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -30,7 +30,8 @@ private const val DEFAULT_PARAMETER = "default"
 private const val DEFAULT_SEPARATOR = ":"
 private const val ROUTE_TO_PARAMETER = "routeTo"
 private const val ROUTE_TO_SEPARATOR = ","
-
+private const val VERBOSE_PARAMETER = "verbose"
```

Comment:
So, this is not truly a verbose flag, it is one to add routing to the response.  Should it be named differently?